### ISSUE TITLE
Resolve atm_namelist issue at runtime on new_restart_files branch

### ISSUE THREAD

**Issue by joeschoonover** [SWPC-IPE/WAM-IPE#97]
In Pull Request #87, the changes in development were propagated to the new_restart_files branch. The code compiles fine, but at crashes at runtime when reading the atm_namelist file.

Though the details are provided in PR #87, the  output is reproduced here : 

```
forrtl: severe (19): invalid reference to variable in NAMELIST input, unit 35, file /scratch4/NCEPDEV/stmp4/Raffaele.Montuoro/march2013/atm_namelist, line 2, position 34
```

Line 2, position 34 of `atm_namelist` is associated with the parameter `DELTIM`.

**Comment by joeschoonover** [comment]
After the merge, for some reason, DELTIM was removed from the `&nam_dyn` namelist in the file `NEMS/src/atmos/gsm/dyn/compns_dynamics.f`. I'm going to attempt to reintroduce this parameter to the namelist to see if this fixes this issue.

**Comment by joeschoonover** [comment]
This fixed this issue, but now the namelist read fails with 

```
forrtl: severe (19): invalid reference to variable in NAMELIST input, unit 35, file /scratch3/NCEPDEV/swpc/scrub/Joseph.Schoonover/st
mp/Joseph.Schoonover/march2013/atm_namelist, line 25, position 34
```

**Comment by joeschoonover** [comment]
DELTIM also should exist as a namelist parameter in `NEMS/src/atmos/gsm/phys/compns_physics.f`. I'll pull this file from development also

**Comment by joeschoonover** [comment]
Note that the other option is to update our atm_namelist file to not specify DELTIM. However, we'll need to hear from @wanghj2015 on howto correctly specify the time step for WAM

**Comment by joeschoonover** [comment]
Fixed this last namelist issue ( hash id 7a0f375788225e9c005eab77b955e95d76d2b58f ). What's odd is that the new_restart_files branch runs without needing to update our wam initial condition files. I haven't looked at the output, but I thought that we had a problem reading in the WAM initial conditions before. At any rate, we can close this issue and carry on testing this branch in Issue #33
